test/browser/render.test.js

```diff
@@ -2013,4 +2013,11 @@ describe('render()', () => {
 		render(<App />, scratch);
 		expect(scratch.innerHTML).to.equal('hello world');
 	});
+	it('should hydrate <template> tags ', () => {
+		const App = () => <template><h1>it works</h1></template>		
+		scratch.innerHTML = `<template><h1>it works</h1></template>`
+
+		render(<App />, scratch);
+		expect(scratch.innerHTML).to.equal(`<template><h1>it works</h1></template>`)
```

Comment:
Not sure if you intentionally skipped the precommit hook, but this would pop up the next time anyone else tried to edit the file w/ hooks enabled.

```suggestion
		const App = () => <template><h1>it works</h1></template>;
		scratch.innerHTML = `<template><h1>it works</h1></template>`;

		render(<App />, scratch);
		expect(scratch.innerHTML).to.equal(`<template><h1>it works</h1></template>`);
```

Reply:
No worries! I do that myself _a lot_